packages/monitor-v2/src/llm-bot/common.ts

```diff
@@ -173,6 +173,53 @@ export class OptimisticOracleRequest {
   }
 }
 
+/**
+ * Interface representing the additional data fields for a disputable oracle request.
+ */
+interface DisputableData {
+  correctAnswer: boolean;
+  rawLLMInput: string;
+  rawLLMOutput: string;
```

Comment:
I could imagine the LLM strategies involving more than single string input and output, but I doubt much of the implementation will depend on this, so we can always change later.